incubator/swift/README.md

```diff
@@ -35,4 +35,34 @@ Templates are used to create your local project and start your development. When
     Hello, world!
     ```
 
-**NOTE:** Currently the `appsody deploy` command only works for deploying web applications.
\ No newline at end of file
+**NOTE:** Currently the `appsody deploy` command only works for deploying web applications.
+
+## Enabling an existing Project
+
+The Swift stack can be used with an existing server-side Swift project in order to provide an interative containerized development and test environment, and to "cloud package" it into an optimized production Docker container.
+
+You can enable an existing project as follows:
+
+1. Go to you project directory, e.g.:
+
+    ```bash
+    cd my-project
+    ```
+    
+2. Initialise your an Appsody project with the Swift stack, but without a template:
```

Comment:
nit: Initialise **->** Initialize

Reply:
remove **an**
Initialize your Appsody project with the Swift stack, but without a template:

Reply:
Fixed.